src/pages/members/index.astro

```diff
@@ -0,0 +1,19 @@
+---
+import Layout from '~/layouts/MembersLayout.astro';
+import '~/assets/styles/custom.css';
+import { staffMembers } from '~/data/mockStaffMembers';
+import StaffCard from '~/components/widgets/StaffCard.astro';
+
+const metadata = {
+  title: 'Persian Flutter | Members',
+  ignoreTitleTemplate: true,
+};
+---
+
+<Layout metadata={metadata}>
```

Comment:
Rename the Layout Component to MembersLayout to make it more clear. I guess you just need to update Import name